src/backend/tests/stats/test_stats_save_result.py

```diff
@@ -0,0 +1,57 @@
+from flask import g
+
+import expungeservice
+from expungeservice.database.db_util import rollback_errors
+from expungeservice.stats import save_result
+from tests.factories.crawler_factory import CrawlerFactory
+from expungeservice.database import get_database
+from tests.endpoints.endpoint_util import EndpointShared
+
+
+class TestStats():
+
+    def test_save_result(self):
+
+        with expungeservice.create_app("development").app_context():
+
+            expungeservice.request.before()
+            database = get_database()
+            user_id = "user_id"
+            request_data = {
+                "first_name":"first_name",
+                "last_name":"last_name",
+                "middle_name":"middle_name",
+                "birth_date":"birth_date",
+            }
+
+
+            record = CrawlerFactory.create(CrawlerFactory.setup())
+
+            save_result(user_id, request_data, record)
+
+            expungeservice.request.after(None)
+
+            hashed_search_params = hash(
+                user_id +
+                "first_name" +
+                "last_name" +
+                "middle_name" +
+                "birth_date")
```

Comment:
I would prefer if realistic values were used as on the initial read I got confused why you were hashing the field names (when in fact you are hashing values that just happened to be the same as the field names).